### PR TITLE
Added boring model as a ipynb so it can be updated

### LINKED ISSUE #10490: BoringModel notebook import outdated
## 🐛 Bug

The model notebook at GColab you link when creating an issue is outdated. Tries to import metrics from pytorch lightning:  

```python
from pytorch_lightning.metrics.functional import accuracy
```

I believe this is because pytorch_lightning doesn't include torchmetrics anymore, and they are a completely separate library.

### To Reproduce

Go to the link and try to run it: 
https://colab.research.google.com/drive/1HvWVVTK8j2Nj52qU4Q4YCyzOm0_aLQF3?usp=sharing

### Expected behavior

To run smoothly!

### How to solve it

Replace the import with a torchmetrics import:
```python
from torchmetrics.functional import accuracy
```

I would open a PR myself but I couldnt find the notebook on the repo. It could be beneficial maybe to include it as part of the `pl_examples`? Then link the colab notebook to the version on master branch. That I could open a PR :) 

### Environment
Latest google colab

### PR DESCRIPTION
**Pull request by jiwidi**
## What does this PR do?
Fixes #10490

This PR adds the boring model notebook along with a needed fix on an outdated import. The purpose of this PR is so the google colab linking to the boring model aims at the ipynb version on master, allowing the PL team to update and mantain the notebook within github.

If this PR moves forward, the default issue link should be updated.

## Before submitting

- [x] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [ ] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [x] Did you verify new and **existing tests pass** locally with your changes?
- [n/a] Did you list all the **breaking changes** introduced by this pull request?
- [Is it necessary? Could do it needed] Did you **update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)**? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [ ] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [ ] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Yes

### CODE REVIEW

**Review by akihironitta — APPROVED** [review]

File: pl_examples/The_BoringModel.ipynb [hunk 1]
```diff
@@ -0,0 +1,344 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "name": "The BoringModel.ipynb",
```
Comment (carmocca) [inline pl_examples/The_BoringModel.ipynb:6]: You can also update the default issue link here: it's in

https://github.com/PyTorchLightning/pytorch-lightning/blame/master/.github/ISSUE_TEMPLATE/bug_report.md#L19

**Comment by rohitgr7** [comment]
1. Shall we remove the torchmetrics from the bug report model? The reason to remove torchmetrics is we don't want user to explicitly install it always since its a completely separate package and also `accuracy` imported isn't used anywhere.
2. Shall we remove bolts requirements? Whatever's imported from bolts isn't used either.
3. Keep both notebook and script under pl_examples/bug_report/<.py and .ipynb>??

**Comment by tchaton** [comment]
> * Shall we remove the torchmetrics from the bug report model? The reason to remove torchmetrics is we don't want user to explicitly install it always since its a completely separate package and also `accuracy` imported isn't used anywhere.
> * Shall we remove bolts requirements? Whatever's imported from bolts isn't used either.
> * Keep both notebook and script under pl_examples/bug_report/<.py and .ipynb>??

We should also drop torch vision, Mnist import. Let's keep it minimal.

**Comment by rohitgr7** [comment]
yes, I'd say just copy paste stuff from here in their respective cells: https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pl_examples/bug_report_model.py

**Comment by jiwidi** [comment]
I'll change the imports again and add the colab link to the github issue template then!

File: pl_examples/The_BoringModel.ipynb (hunk 1 above)
Comment (jiwidi) [reply pl_examples/The_BoringModel.ipynb]: Hi! 

I updated the link with a path to the master path of the notebook. The link wont work as there is not a notebook there yet. We could merge like this or leave it to a second PR and its properly tested

**Review by carmocca — APPROVED** [review]
LGTM

For any reviewers, you can check the Notebook with this link:
https://colab.research.google.com/github/PytorchLightning/pytorch-lightning/blob/master/pl_examples/bug_report/The_BoringModel.ipynb

**Review by tchaton — APPROVED** [review]
LGTM ! Much cleaner !

**Review by Borda — APPROVED** [review]

**Pull request merged**